src/js/actions/policy.js

```diff
@@ -320,29 +320,21 @@ define(function (require, exports) {
      * @return {Promise}
      */
     var beforeStartup = function () {
-        var keyboardMode = adapterUI.keyboardPropagationMode.NEVER_PROPAGATE,
-            keyboardDescriptor = {
-                defaultMode: keyboardMode
-            },
-            keyboardModePromise = adapterUI.setKeyboardPropagationMode(keyboardDescriptor)
-                .bind(this)
-                .then(function () {
-                    this.dispatch(events.policies.MODE_CHANGED, {
-                        kind: PolicyStore.eventKind.KEYBOARD,
-                        mode: keyboardMode
-                    });
-                }),
-            pointerMode = adapterUI.pointerPropagationMode.ALPHA_PROPAGATE,
+        var defaultKeyboardMode = adapterUI.keyboardPropagationMode.NEVER_PROPAGATE,
+            keyboardModePromise = this.transfer(setMode, PolicyStore.eventKind.KEYBOARD,
+                defaultKeyboardMode),
+            defaultPointerMode = adapterUI.pointerPropagationMode.ALPHA_PROPAGATE,
             // Alpha is the default pointer mode, so we don't need to change it here
```

Comment:
I think when I wrote this I didn't realize we were actually changing the mode dynamically. I wonder if we should set it here even though it is the "default" in case, e.g., there is a controller error and we reset while in a non-default mode. What do you think?